code/modules/admin/verbs/playsound.dm

```diff
@@ -57,10 +70,16 @@ GLOBAL_LIST_EMPTY(sounds_cache)
 	var/list/sounds = file2list("sound/serversound_list.txt")
 	sounds += GLOB.sounds_cache
 
-	var/melody = input("Select a sound from the server to play", "Server sound list") as null|anything in sounds
+	var/sound/melody = input("Select a sound from the server to play", "Server sound list") as null|anything in sounds
 	if(!melody)	return
 
-	play_sound(melody)
+	log_admin("[key_name(src)] played server sound [melody]")
+	message_admins("[key_name_admin(src)] played server sound [melody]", 1)
+
+	for(var/mob/M in GLOB.player_list)
+		if(prefs.sound & SOUND_LOBBY)
```

Comment:
Is the `SOUND_LOBBY` here intentional? All of the others are checking for `SOUND_MIDI`, which makes a lot more sense to me.
Also this is checking for the preferences of `src`, which is the admin in this case.
```suggestion
		if(M.client.prefs.sound & SOUND_MIDI)
```

Reply:
This one only plays the lobby music and two? Three? other sounds. With is being lobby music, i figured use sound_lobby over sound_midi.

Reply:
That makes sense. I'd still suggest having it check the midi preference anyway since it *is* being played by an admin.
Something like this instead:
```suggestion
		if((M.client.prefs.sound & SOUND_LOBBY) && (M.client.prefs.sound & SOUND_MIDI))
```